### PR TITLE
feat(entity): add clear method which resets dirty properties

### PR DESCRIPTION
**Pull request by Kukks**
… values
for #191

### CODE REVIEW

File: src/entity.js [hunk 1]
```diff
@@ -394,7 +394,19 @@ export class Entity {
...
     return this.markClean();
   }
+  /**
+   * Sets the entity's properties to their clean values
+   *
+   * @return {Entity} itself
+   * @chainable
+   */
+  clear() {
+    if(!this.isNew()) {
```
Comment (RWOverdijk) [inline src/entity.js:404]: `if () {}` (add a space after if)

**Review by RWOverdijk — CHANGES_REQUESTED** [review]
Thanks for the PR. Almost perfect!

Could you take out the build please? You only have to PR the actual change, we run the build on every release through our release chain.

**Comment by RWOverdijk** [comment]
Last micro note: could you change the commit message to:

`feat(entity): add clear method which resets dirty properties`

**Comment by Kukks** [comment]
@RWOverdijk Is there an easy way to doing this other than re-creating the PR with a new branch with the changes copied over? I'm not that skilled in git. :)

**Comment by RWOverdijk** [comment]
@Kukks Ehm, easy way no. It's not a lot of code, so maybe you could even edit it in the browser. Otherwise I can do it for you, but that will be tomorrow :)

**Comment by Kukks** [comment]
Opened up a new PR without any of the added build files and whatnot. :)

**Pull request closed without merging**